hudi/assets/dashboards/overview.json

```diff
@@ -688,7 +688,6 @@
                     }
                 ]
             },
-            "id": 3431139800551312,
```

Comment:
This looks like the ID of a widget rather than the ID of the outer dashboard object; is removing this expected?

Reply:
Good catch! It's the only one I got to update manually but I failed to do it properly 🙈
Addressed in [#5f70961](https://github.com/DataDog/integrations-core/pull/14409/commits/5f7096142efd2c7d9ff0ba46a674604298c08e97)